lib/main.js

```diff
@@ -3,7 +3,7 @@ const UTIL = require('./util.js');
 const QS = require('querystring');
 
 const main = module.exports = (searchString, options, callback) => { // eslint-disable-line consistent-return
-  // Check wether options wether no options were provided
+  // Check weather options weather no options were provided
```

Comment:
Guess it should actually be
`// Check whether options where provided`

Reply:
yep